src/main/java/org/zalando/planb/revocation/config/StorageConfig.java

```diff
@@ -42,13 +43,25 @@ public RevocationStore revocationStore() {
 
         // Defaults to in-memory, when CassandraProperties are absent;
         if (StringUtils.isEmpty(cassandraProperties.getContactPoints())) {
-            return new InMemoryStore();
+            return new InMemoryRevocationStore();
         }
 
-        return new CassandraStore(cassandraSession(), cassandraProperties.getReadConsistencyLevel(),
+        return new CassandraRevocationStore(cassandraSession(), cassandraProperties.getReadConsistencyLevel(),
                 cassandraProperties.getWriteConsistencyLevel(), cassandraProperties.getMaxTimeDelta());
     }
 
+    @Bean
+    public AuthorizationRulesStore authorizationRulesStore() {
+
+        // Defaults to in-memory, when CassandraProperties are absent;
+        if (StringUtils.isEmpty(cassandraProperties.getContactPoints())) {
+            return new InMemoryAuthorizationRuleStore();
+        }
+
+        return new CassandraAuthorizationRuleStore(cassandraSession(), cassandraProperties.getReadConsistencyLevel(),
```

Comment:
Maybe we should avoid a duplicate call to `cassandraSession()`, but instead expose the session as Spring bean and re-use it everywhere.